src/externalized/AmmoTypeModel.cc

```diff
@@ -21,8 +21,7 @@ void AmmoTypeModel::serializeTo(JsonObject &obj) const
 AmmoTypeModel* AmmoTypeModel::deserialize(JsonObjectReader &obj)
 {
 	int index = obj.GetInt("index");
-	ST::string internalName = obj.GetString("internalName");
-	return new AmmoTypeModel(index, internalName);
+	return new AmmoTypeModel(index, obj.GetString("internalName"));
```

Comment:
There's no need to explicitly use std::move here?

Reply:
I thought so too, but [Compiler Explorer](https://godbolt.org/z/539Yccqn3) proved me wrong. 84 lines with the move, 106 without it. C++ move semantics are complicated, that's for sure.

Reply:
Oh, I see, I misunderstood your question. The std::move has to be in the member initialization list. Compiler Explorer does not show any difference between the constructor calls F(c) and F(std::move(c)).

Reply:
Ah right, it already is.